apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -395,8 +399,33 @@ const SwapForm = () => {
 
   const sourceAccount = swapTransaction.swap.from.account;
   const sourceCurrency = swapTransaction.swap.from.currency;
+  const sourceParentAccount = swapTransaction.swap.from.parentAccount;
+  const targetAccount = swapTransaction.swap.to.account;
+  const targetParentAccount = swapTransaction.swap.to.parentAccount;
   const targetCurrency = swapTransaction.swap.to.currency;
 
+  // We check if a decentralized swap is available to conditionnaly render an Alert below.
+  // All Ethereum related currencies are considered available
+  const showDEXLinkBanners = useMemo(() => {
+    // if we are showing DEX quotes, we don't want to show the link banners
+    if (showDexQuotes) {
+      return false;
+    }
+
+    if (sourceAccount && targetAccount) {
+      const sourceMainAccount = getMainAccount(sourceAccount, sourceParentAccount);
+      const targetMainAccount = getMainAccount(targetAccount, targetParentAccount);
+
+      if (
+        targetMainAccount.currency.family === "ethereum" &&
```

Comment:
We can move this logic to common when we update llm, but could we support binance and polygon family here too?

const dexFamilyList = ["ethereum", "binance", "polygon"];

Reply:
Good call https://github.com/LedgerHQ/ledger-live/pull/2123/commits/27c406c07719c84d236fd8860041b30b9d5d578a